tests/test_obdsim.py

```diff
@@ -32,26 +34,25 @@ def good_rpm_response(r):
            (r.value >= 0.0 * Unit.rpm)
 
 
-@pytest.mark.skipif(not pytest.config.getoption("--port"),
```

Comment:
`pytest.mark.skipif` went obsolete and is not available in pytest v.4 anymore, that's why I rewritten its functionality as a fixture.